Sources/Public/Animation/AnimationView.swift

```diff
@@ -803,12 +819,22 @@ final public class AnimationView: AnimationViewBase {
       animationLayer.add(positionAnimation, forKey: positionKey)
       animationLayer.add(xformAnimation, forKey: transformKey)
     } else {
-      CATransaction.begin()
-      CATransaction.setAnimationDuration(0.0)
-      CATransaction.setAnimationTimingFunction(CAMediaTimingFunction(name: .linear))
-      animationLayer.position = position
-      animationLayer.transform = xform
-      CATransaction.commit()
+      // In performance tests, we have to wrap the animation view setup
+      // in a `CATransaction` in order for the layers to be deallocated at
+      // the correct time. The `CATransaction`s in this method interfere
+      // with the ones managed by the performance test, and aren't actually
+      // necessary in a headless environment, so we disable them.
+      if TestHelpers.performanceTestsAreRunning {
```

Comment:
I also had to do this in https://github.com/airbnb/lottie-ios/pull/1518/files#diff-b8031f1d50b5687c758122fa8aa487e09cb9068eee0db0d2ecff606ba944e9d3R834. I must have missed this on the last PR, or changes the code path just enough to start hitting this in the performance tests.